### ISSUE TITLE
UnhandledPromiseRejectionWarning when serving fresh installation

### ISSUE THREAD

**Issue by ryan7273** [pattern-lab/patternlab-node#1231]
I am using Pattern Lab Node `v14.7.0` on `Linux`, with Node `v14.7.0`, using a `Vanilla` Edition.

##### Expected Behavior
It should build everything and run the server without generating errors or warnings.

##### Actual Behavior
```
Pattern Lab is being served from http://127.0.0.1:3000
(node:201) UnhandledPromiseRejectionWarning: Error: Exited with code 3
    at ChildProcess.<anonymous> (/root/node_modules/opn/index.js:84:13)
    at Object.onceWrapper (events.js:421:26)
    at ChildProcess.emit (events.js:314:20)
    at maybeClose (internal/child_process.js:1051:16)
    at Socket.<anonymous> (internal/child_process.js:442:11)
    at Socket.emit (events.js:314:20)
    at Pipe.<anonymous> (net.js:673:12)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:201) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:201) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
The server does run and it looks like everything works as expected, but the deprecation warning makes me nervous. I don't want to be unable to upgrade node later without breaking Pattern Lab.

##### Steps to Reproduce

1. docker container run -ti --rm node:14.7.0 /bin/bash
1. cd
1. npm create pattern-lab
    1. ./
    1. Handlebars
    1. Handlebars base patterns (some basic patterns to get started with)
    1. Y
1. npm run pl:build
1. npm run pl:serve

**Comment by pcate-fls** [comment]
I've run into the same warning when creating a fresh pattern library. In my case it was with the twig engine, but otherwise the steps are the same.

**Comment by JosefBredereck** [comment]
The issue was released with v5.13.0 🎉
